src/controllers/focus-lost-controller.stories.js

```diff
@@ -0,0 +1,97 @@
+import { css, html, LitElement, render } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { enhanceStoriesNames } from '../stories/lib/story-names.js';
+import { FocusLostController } from './focus-lost-controller.js';
+import docsPage from './focus-lost-controller.md';
+import '../components/cc-button/cc-button.js';
+
+class MyList extends LitElement {
+  static get properties () {
+    return {
+      elements: { type: Array },
+    };
+  }
+
+  constructor () {
+    super();
+    this.elements = [];
+
+    // eslint-disable-next-line no-new
+    new FocusLostController(this, 'div', ({ suggestedElement }) => {
+      suggestedElement?.querySelector('cc-button').focus();
+    });
+  }
+
+  _onRemove (event) {
+    this.elements = this.elements.filter((e) => e !== event.currentTarget.dataset.name);
+  }
+
+  focus () {
+    this.shadowRoot.querySelector('cc-button').focus();
+  }
+
+  firstUpdated (_changedProperties) {
+    setTimeout(() => this.focus());
+  }
+
+  render () {
+    return repeat(this.elements, (e) => e, (e) => {
+      return html`
+        <div>
+          <cc-button @cc-button:click=${this._onRemove} data-name="${e}">Remove</cc-button>
+          <span>${e}</span>  
+        </div>
+      `;
+    });
+  }
+
+  static get styles () {
+    return [
+      // language=CSS
+      css`
+        div {
+          margin-bottom: 0.2em;
+        }
+      `,
+    ];
+  }
+}
+window.customElements.define('cc-list', MyList);
```

Comment:
I would name the tag `my-list`, this way it's clear it's an example, it's named like the class and won't be clashing with a future list component.